util/src/overlaydb.rs

```diff
@@ -192,6 +192,7 @@ impl HashDB for OverlayDB {
 }
 
 #[test]
+#[cfg_attr(feature="dev", allow(blacklisted_name))]
```

Comment:
So the function is using names like `foo` and `bar` which are blacklisted by default: https://github.com/Manishearth/rust-clippy/wiki#blacklisted_name

I could rewrite the test to use some different names, but not sure if it's worth it. If it happens in the future I will disable this lint completely.